zuul.d/tempest_multinode.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 - job:
-    name: cifmw-base-multinode-tempest
+    name: cifmw-base-multinode-antelope-tempest
```

Comment:
Did you double check if this change will not break other jobs? You are renaming a base job here..

Reply:
@arxcruz in the upstream, its not breaking anything. https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-framework%20cifmw-base-multinode-tempest&type=code

Reply:
in the downstream also, we are not consuming this job - https://code.engineering.redhat.com/gerrit/gitweb?p=openstack%2Ftripleo-ci-internal-jobs.git&a=search&h=HEAD&st=grep&s=cifmw-base-multinode-tempest